Makefile.prow

```diff
@@ -0,0 +1,15 @@
+# Copyright Contributors to the Open Cluster Management project
+
+-include /opt/build-harness/Makefile.prow
+
+
+.PHONY: lint
+lint:
+	GOPATH=$(go env GOPATH)
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${GOPATH}/bin" v1.47.1
+	CGO_ENABLED=0 GOGC=25 golangci-lint run --timeout=3m
+	gosec ./...
+
+.PHONY: unit-test
+unit-test:
+	go test ./... -v -coverprofile cover.out
```

Comment:
Makefile also does not need lint and unit-test targets , this need the script to populate the right images - and when we have the e2e test , a target for that.

Reply:
Removed unit-test and lint targets.